include/exadg/operators/inverse_mass_operator.h

```diff
@@ -28,9 +28,24 @@
 
 // ExaDG
 #include <exadg/matrix_free/integrators.h>
+#include <exadg/operators/mass_operator.h>
+#include <exadg/solvers_and_preconditioners/preconditioners/block_jacobi_preconditioner.h>
 
 namespace ExaDG
 {
+struct InverseMassOperatorData
+{
+  unsigned int dof_index;
+  unsigned int quad_index;
```

Comment:
Let's also initialize these two parameters explicitly in this struct. The default value is `0`.